### PR TITLE
Create pt-BR.yml

### PR DESCRIPTION
**Pull request by rhiroshi**
Adding brazillian portuguese translations

### CODE REVIEW

File: config/locales/pt-BR.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+pt-BR:
+  errors:
+    messages:
+      taken_in_past: 'Foi usada anteriormente.'
+      equal_to_current_password: 'Deve ser diferente da senha atual.'
+      password_complexity:
+        digit:
+          one: Deve conter ao menos um digito
```
[the remark this answers is not in the document]
Comment (rhiroshi) [reply config/locales/pt-BR.yml:8]: Yes, thats correct. Sorry about those typos!

File: config/locales/pt-BR.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          other: Deve conter ao menos %{count} digitos
```
[the remark this answers is not in the document]
Comment (rhiroshi) [reply config/locales/pt-BR.yml:9]: Yes, thats correct. Sorry about those typos!

File: config/locales/pt-BR.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+pt-BR:
```
[the remark this answers is not in the document]
Comment (rhiroshi) [reply config/locales/pt-BR.yml:1]: I see. I was wondering if i should append -BR to the key aswel but i tought people would confuse with Portugal portuguese as i've never seen pt-PT. I have no opposition to drop the -BR as Rails do support [fallbacks](https://guides.rubyonrails.org/v5.2.0/configuring.html#configuring-i18n)

File: config/locales/pt-BR.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+          other: Deve conter ao menos %{count} digitos
+        lower:
+          one: Deve conter ao menos uma letra minúscula
+          other: Deve conter ao menos %{count} letras minúsculas
+        symbol:
+          one: Deve conter ao menos um símbolo
+          other: Deve conter ao menos %{count} símbolos
+        upper:
+          one: Deve conter ao menos uma letra maiúscula
+          other: Deve conter ao menos %{count} letras maiúsculas
```
[the remark this answers is not in the document]
Comment (rhiroshi) [reply config/locales/pt-BR.yml]: Sure! When i was translating the file i wondered why it was written in lowercase, but after i implemented the functions and i got to see the actual messages i understood. Will fix!

[the remark this answers is not in the document]
Comment (tmr08c) [reply config/locales/pt-BR.yml]: Glad it makes sense :) 

Thank you!

File: config/locales/pt-BR.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (tmr08c) [reply config/locales/pt-BR.yml:1]: Thank you!  I think starting with `pt` and revisiting if someone wants to implement Portugal Portuguese will be a good start.

**Comment by rhiroshi** [comment]
@tmr08c done!

**Comment by tmr08c** [comment]
Thank you for your contribution, @rhiroshi!

**Pull request merged**